scss/_components.scss

```diff
@@ -8,8 +8,88 @@
     }
 
     &:hover {
-        svg path{
+        svg path {
             fill: $star-icon-hover-bg
         }
     }
+}
+
+.logo-wrapper {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    h1 {
+        margin-left: 5px;
+        letter-spacing: 1px;
+        font-weight: 700;
+    }
+}
+
+.favorite-city-dropdown {
+    width: 300px;
+    height: 45px;
+    padding: 0.5rem;
```

Comment:
If you don't know, it's good to know what case you should use px or rem for accessibility. If you already know, it's perfect :)
https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/